tests/orgs/toggleActive/endpoint.test.js

```diff
@@ -4,76 +4,177 @@ var test = require('tape');
     if (err) return t.fail('error initialising server');
 
-    server.inject({
-      method: 'GET',
-      url: '/orgsToggleActive?id=8'
-    }, function (res) {
-
-      t.deepEqual(res.result, [], 'successful enabling of org returns an array');
-
-      server.inject({
-        method: 'GET',
-        url: '/orgsGetDetails?id=8'
-      }, function (res) {
-        t.ok(res.result.org.active, 'the toggle active function has enabled the previously inactive org');
-
-        t.end();
-        pool.end()
-        server.stop()
+    server.inject(orgDetails(orgId), function (res) {
+      t.ok(
+        !res.result.org.active,
+        'the org is inactive to begin with'
+      );
+      server.inject(allUsers, function (res) {
+        preToggleUsers = res.result;
+        server.inject(orgsToggleActive(orgId), function (res) {
+          t.deepEqual(
+            res.result,
+            [],
+            'successful disabling of org returns an empty array'
+          );
+          server.inject(orgDetails(orgId), function (res) {
+            t.ok(
+              res.result.org.active,
+              'previously inactive org has been enabled'
+            );
+            server.inject(allUsers, function (res) {
+              t.deepEqual(
+                res.result,
+                preToggleUsers,
+                'none of the users have been affected by change in org activity'
+              );
+              t.end();
+              pool.end()
+              server.stop()
+            });
+          });
+        });
       });
     });
   });
 });
 
 
-// test disbable org with wrong id -> error?
-test('enable an inactive org', function (t) {
+// test inactive org with active user
+test('enable an inactive org with an active primary user', function (t) {
+  var orgId = 8;
+  var userId = 11;
   init(config, function (err, server, pool) {
     if (err) return t.fail('error initialising server');
 
-    server.inject({
-      method: 'GET',
-      url: '/orgsToggleActive?id=10000'
-    }, function (res) {
-
-      t.equal(res.statusCode, 404, 'Update an organisation that doest exist returns a 404');
+    server.inject(orgDetails(orgId), function (res) {
+      t.ok(
+        !res.result.org.active,
+        'the org is inactive to begin with'
+      );
+      server.inject(userDetails(userId), function (res) {
+        t.ok(
+          res.result[0].active,
+          'the user is active to begin with'
+        );
+        server.inject(orgsToggleActive(orgId), function (res) {
+          t.deepEqual(
+            res.result,
+            [],
+            'successful enabling of org returns an empty array'
+          );
+          server.inject(orgDetails(orgId), function (res) {
+            t.ok(
+              res.result.org.active,
+              'previously inactive org is now enabled'
+            );
+            server.inject(userDetails(userId), function (res) {
+              t.ok(
+                !res.result[0].active,
```

Comment:
this suggests that the updated user is NOT active, which contradicts what is supposed to happen according to the comment.